plugins/woocommerce/tests/e2e-pw/tests/merchant/settings-tax.spec.js

```diff
@@ -29,7 +29,7 @@ test.describe( 'WooCommerce Tax Settings > enable', () => {
 	} );
 } );
 
-test.describe( 'WooCommerce Tax Settings', () => {
+test.describe.serial( 'WooCommerce Tax Settings', () => {
```

Comment:
Tests in this spec file rely heavily on the earlier ones being successful, so we need to add `serial` here.